Makefile

```diff
@@ -270,3 +270,14 @@ golangci: get-ci-tools
 # Run go lint against code
 golint: get-ci-tools
 	PATH=$(GOBIN):$(PATH); $(CI_TOOLS_REPO_DIR)/test-runner/golint.sh
+
+.PHONY: gowork
+gowork: ## Generate go.work file to support our multi module repository
+	test -f go.work || go work init
+	go work use .
+	go work use ./api
+
+.PHONY: operator-lint
+operator-lint: gowork ## Runs operator-lint
+	GOBIN=$(LOCALBIN) go install github.com/gibizer/operator-lint@latest
```

Comment:
probably better to pin this to a specific version to avoid CI breakage if a new version is tagged from this that introduce new checks that might fail for the current repo. If latest works then an equivalent would be the last tagged one: `github.com/gibizer/operator-lint@v0.2.1`

Reply:
Makes sense. I can change this but we do we use latest elsewhere I guess.

Reply:
Yeah when I first introduced operator-lint I used @latest. Then I realized that it means whatever new check is introduced in operator-lint can break our CI until the code is adjusted to the new check. So I tried to go through the repos and pin the version but probably missed some.